runtime/interpreter/value.go

```diff
@@ -1638,7 +1645,13 @@ func (v *ArrayValue) RecursiveString(seenReferences SeenReferences) string {
 func (v *ArrayValue) Append(interpreter *Interpreter, getLocationRange func() LocationRange, element Value) {
 
 	// length increases by 1
-	common.UseMemory(interpreter, common.NewArrayAdditionalLengthUsage(int(v.array.Count()), 1))
+	dataSlabs, metaDataSlabs := common.AdditionalAtreeMemoryUsage(
```

Comment:
Don't we need to incorporate elementOverhead for arrays, similar to dictionaries/composites?

Reply:
No, according to @fxamacker's comment here: https://github.com/onflow/flow-go/pull/2357#discussion_r862923066 the overhead is only for the key/value pair of atree OrderedMaps, and thus arrays do not have the same overhead cost per element.